src/core/extra.ml

```diff
@@ -297,8 +299,8 @@ module Array =
       let aux (el, er) (accl, accr) = (el :: accl, er :: accr) in
       Array.fold_right aux a ([], [])
 
-    (** [drop n a] discards the first [n] elements of [a].  The empty array is
-        returned if [n > length a]. *)
+    (** [drop n a] discards the first [n] elements of [a].
+        The empty array is returned if [n > length a]. *)
```

Comment:
Line break

Reply:
For me the first version is easier to write than this one, but OK.